BDArmory/Weapons/ModuleWeapon.cs

```diff
@@ -6180,6 +6192,7 @@ public override string GetInfo()
                 {
                     output.AppendLine($"Rounds Per Minute: {roundsPerMinute * (fireTransforms?.Length ?? 1)}");
                     if (SpoolUpTime > 0) output.AppendLine($"Weapon requires {SpoolUpTime} seconds to come to max RPM");
+                    output.AppendLine($"Shot Deviation: {(((Mathf.Sin(maxDeviation / 57.29577f) * 1000) / 2) * 1.285f) * 2:F2} mrad, 80% hit");
```

Comment:
Technically, this should be `Mathf.Tan(Mathf.Deg2Rad * maxDeviation) * 1000 * (1.285f / 2) * 2`, but the difference is only about 1e-5 for typical `maxDeviation` values.